devex/vscode-extension/README.md

```diff
@@ -0,0 +1,383 @@
+# Open Enclave extension for Visual Studio Code
+
+A
+[Visual Studio Code](https://code.visualstudio.com/)
+[extension](https://marketplace.visualstudio.com/VSCode)
+supporting
+[Open Enclave](https://openenclave.io/sdk/),
+including development, debugging, emulators, and deployment!
+
+For more information on the technology,
+[see here](https://github.com/openenclave/openenclave).
+
+Currently, this extension supports the following combinations of project types,
+and operating system and trusted execution environments:
+
+1. Linux Development Machine:
+   1. Standalone Project Targets:
+      1. Linux Host w/. Intel SGX Enclave
```

Comment:
nit: would be better to spell out "with" which is only 1 character longer than "w/." and is more understandable by non-native English speakers.

Reply:
Changed.